### ISSUE TITLE
Win11 right click context menu LOST!

### ISSUE THREAD

**Issue by CharlesXSong** [stefankueng/BowPad#269]
The context menu 'Open Folder / Edit with BowPad ' exists in [Version 2.8.2](https://github.com/stefankueng/BowPad/releases/tag/2.8.2), disappears in [Version 2.8.3](https://github.com/stefankueng/BowPad/releases/tag/2.8.3)😭

**Comment by stefankueng** [comment]
works fine for me

**Comment by CharlesXSong** [comment]
As u can see, the context is in the 2-level menu but not the 1-level
![image](https://user-images.githubusercontent.com/50441285/179660280-0d0df8b6-3cb6-40a8-9b03-44d623f4b614.png)
![image](https://user-images.githubusercontent.com/50441285/179661053-7eff3f9e-332c-4084-a60e-36333208744f.png)

**Comment by stefankueng** [comment]
it's not that I don't believe you. It's just that I have no idea why it doesn't work for you.
I've tried now on two workstations, one with official Win11 and one with the Win11 beta channel version. It works on both.

Can you try uninstall, reboot, install, reboot and check if maybe that fixes your problem?

**Comment by CharlesXSong** [comment]
> Can you try uninstall, reboot, install, reboot and check if maybe that fixes your problem?

I tried to uninstall, reboot, reboot and reinstall, but they didn't solve the problem. /(ㄒoㄒ)/~~
***
Then I tried to use "run as administrator" to install but found that the '.msi' can only use the default installation method.
![image](https://user-images.githubusercontent.com/50441285/179912952-de949b2f-7785-4717-b251-2657b4b37b24.png)
Fortunately, after an usual and ineffective installation, I took a shot in the dark to click the second option “修复(P)” means 'repair'.
Guess what?!!! The first level right-click menu 'Open Folder / Edit with BowPad' appears!!!!!!
![image](https://user-images.githubusercontent.com/50441285/179913799-06e70a85-8ccc-4d67-b819-ec2c63907cfd.png)
Ohhhh🎊~ I'm so happy to see its return!!!😄
***
Thank you very much for your reply and patience!
Additionally, really appreciate your BowPad. It's a soooooo excellent software.😁
